lib/packages/docutils/rstast.nim

```diff
@@ -77,6 +79,8 @@ type
                               ## the document or the section; and rnEnumList
                               ## and rnAdmonition; and rnLineBlockItem
     level*: int               ## valid for headlines/overlines only
+    order*: int               ## footnote order (for auto-symbol footnotes and
```

Comment:
Can't use `level` too here? level is currently only used for headlines and overlines.

Reply:
I can of course.

But what's the purpose? Isn't it too blatant to reuse other names for different unrelated goals?

I expected to move in the opposite direction in the future and split also `text` field into 2 fields `text` and `info` to avoid confusion.

Reply:
Memory consumption is always an issue. If you don't like to re-use fields, make the object contain a ``case``. :-) Can be done in a followup PR, of course.